era5cli/utils.py

```diff
@@ -211,14 +212,17 @@ def strtobool(value: str) -> bool:
 def assert_outputfiles_not_exist(outputfiles: List[str]) -> None:
     """Check if files already exist, and prompt the user if they do."""
     if any(Path(file).exists() for file in outputfiles):
-        answer = input(
-            "\n  Some file(s) that will be downloaded already exist in this folder."
-            "\n  Do you want to overwrite them? (Y/N)"
-            "\n  Tip: to skip this flag, use `--overwrite`."
-            "\n"
-        )
+        answer = "no"  # default answer for non-interactive sessions
+        if sys.stdin.isatty():  # only ask for input if the user can reply.
+            answer = input(
+                "\n  Some file(s) that will be downloaded already exist in this folder."
+                "\n  Do you want to overwrite them? (Y/N)"
+                "\n  Tip: to skip this flag, use `--overwrite`."
```

Comment:
Can you move the tip above the question. Now the cursor to type the answer is not close to the question

Reply:
Also `skip this flag` is confusing to me
```suggestion
                "\n  Tip: to always overwrite use `--overwrite`."
```